lib/main_common.pm

```diff
@@ -222,8 +222,7 @@ sub packagekit_available {
 
 sub is_ltp_test {
     return (get_var('INSTALL_LTP')
-          || get_var('LTP_COMMAND_FILE')
-          || get_var('NVMFTESTS'));
```

Comment:
I imagine NVMF tests will be executed anyway when is_kernel_test() is called.

Reply:
Yes, I just don't want someone to be confused by my commit :(